content/en/docs/Configuration/multi-cluster.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Multi-cluster"
+description: "Configuring Kiali for a multi-cluster mesh."
+---
+
+Kiali has [experimental support for Istio multi-cluster installations]({{< relref "../Features/multi-cluster" >}}).
+
+Before proceeding with the setup, ensure your deployment meets the following requirements.
+
+## Requirements
+
+1. Primary-remote deployment
+
+Only the primary-remote deployment is currently supported.
```

Comment:
I think we should get away from the "primary-remote", or "multi-primary" terminology.  You'll notice that Istio has no mention of these terms anymore, in their Deployment Models section.

I think maybe we should just emphasize our own Deployment models.  Which at the moment is (I think):
- One Kiali per control plane (istiod)
  - No support for external control planes
  - Kiali and istiod on the same cluster
- One Kiali per mesh
  - Mesh can span multiple clusters
- One or more data planes
- One metric store (Prometheus or Thanos)
  - multiple clusters require unified metrics
- One trace store
  - multiple clusters require unified traces

Reply:
Ah, OK, I didn't see how they laid out the Install section.  So I retract what I said.  When I looked at the Deployment Models page I didn't see it.  

Is it true that we don't support Multi-Primary?  We've always said that we do, using a Kiali per control plane.  So I'm a bit afraid that it looks like we're giving up support for Multi-Primary.  Minimally it would seem like we could support it like we always have, just that each Kiali may have a limited view (like in Federation).  But could it not also be like two Primary-Remote installs?